.vscode/launch.json

```diff
@@ -56,6 +48,20 @@
             "skipFiles": [
                 "node_modules/**"
             ],
+        },
+        {
+            "name": "Debug tests",
```

Comment:
This didn't work for me. I got the error:
```
/usr/local/bin/python: No module named pytest
```

I couldn't see it in the requirements at all, do we need to add it there?

Reply:
Thanks, I'll wipe everything and check again as it may have been an extension that installed it!

Reply:
Looks like the python extension installs